WooCommerce/Classes/ViewRelated/Dashboard/Settings/Privacy/PrivacySettingsViewController.swift

```diff
@@ -368,10 +457,24 @@ extension PrivacySettingsViewController: UITableViewDelegate {
 
 // MARK: - Private Types
 //
+
+extension PrivacySettingsViewController {
+    enum Localization {
+        static let tableTitle = NSLocalizedString("We value your privacy. " +
+                                                  "Your personal data is used to optimize our mobile apps, improve security, " +
+                                                  "conduct analytics and marketing activities, and enhance your user experience.",
+                                                  comment: "Main description on the privacy screen.")
+        static let tracking = NSLocalizedString("Tracking", comment: "Title of the tracking section on the privacy screen")
+        static let reports = NSLocalizedString("Reports", comment: "Title of the report section on the privacy screen")
+    }
+}
+
 private struct Constants {
     static let rowHeight = CGFloat(44)
     static let separatorInset = CGFloat(16)
     static let sectionHeight = CGFloat(18)
+    static let headerTitleInsets = UIEdgeInsets(top: 16, left: 16, bottom: 32, right: 16)
```

Comment:
This is just a visual issue, but I noticed that these insets don't consistently align the header text with the rest of the cells in the table (as in the design). It's a minor difference on the iPhone I checked (the text extends slightly into the left margin) but it's very noticeable on iPad:

Design|iPhone|iPad
-|-|-
<img src="https://user-images.githubusercontent.com/8658164/236423027-b3d8a213-14ae-4f5a-aff8-4b5f7ee62d8c.png" width="300px">|<img src="https://user-images.githubusercontent.com/8658164/236423045-a58081d4-5337-44b7-b082-4bcfbd95847c.png" width="300px">|<img src="https://user-images.githubusercontent.com/8658164/236424347-5d0f20d4-ed06-4108-89ca-0e40013678d6.png" width="600px">

If that alignment is acceptable we could leave it as-is. Otherwise, I'm not sure how to add the required (flexible) insets to the header view with the grouped table layout, but another option if we want the insets to match the rest of the table might be to add the header text in a regular table cell with a clear background?

Reply:
Good catch up Rachel, I did not test on iPad 😇.
I'm pining the header content to the readable area, it is not pixel-perfect but it looks good enough!

Portrait | Landscape
---- | ----
<img width="709" alt="portrait" src="https://user-images.githubusercontent.com/562080/236479918-3f0b5976-8a5d-4f2b-9af8-6e12acc0b062.png"> | <img width="995" alt="landscape" src="https://user-images.githubusercontent.com/562080/236479931-b3299191-db0a-4d11-9be7-49a4f47f5189.png">